powerhome/playbook
# [PLAY-714] Converting Time Range Inline kit to TS

**What does this PR do?**
Converting Time Range Inline kit to Typescript.

[Runway Ticket](https://nitro.powerhrg.com/runway/backlog_items/PLAY-714)

**How to test?**
1. Go to Time Range Inline kit on Playbook docs
2. Ensure all the examples still work

#### Checklist:
